magento/magento2
# Add missing translations in Magento_UI

### Description
Add missing translations in UI module taken from `mage/validation.js` file.

### Fixed Issues (if relevant)
1. magento/magento2#5956: Untranslatable string "Please enter the same value again."

### Manual testing scenarios
1. Install dutch language pack.
2. Set locale of frontend to dutch.
3. Create new user on the storefront, Fill both password fields with a different value.
4. String should be translated.

### Contribution checklist
